site/source/user-manual/service-guides/vaultwarden/index.rst

```diff
@@ -4,19 +4,16 @@
 Vaultwarden
 ===========
 
-Vaultwarden is a self-hosted password manager, which means your passwords physically live on your Embassy. Be sure to create backups and keep them safe! If you lose your Embassy or uninstall Vaultwarden, and you have not made a backup, all your passwords will be lost forever.  If you currently have a *hosted* account with Bitwarden, you can export your data from that account and import it to your Vaultwarden service on Embassy using the built-in import/export features in any Bitwarden application.
+Vaultwarden is a self-hosted password manager, which means your passwords physically live on your Start9 Server. Be sure to create backups and keep them safe! If you lose your Server or uninstall Vaultwarden, and you have not made a backup, all your passwords will be lost forever. 
 
-.. note:: The version of Bitwarden server used on the Embassy is a fork called "Vaultwarden."  It functions with all of the apps/integrations that are built for, and still named after, "Bitwarden."  We apologize for any confusion this may cause.
+If you currently have a **hosted** account with Bitwarden, you can export your data from that account and import it to your Vaultwarden service on your Start9 Server using the built-in import/export features in any Bitwarden application!
 
-Check out our `Vaultwarden video <https://www.youtube.com/watch?v=YcxxVHpm9j0>`_ for basic features and setup!
+.. note:: The version of Bitwarden Server used on your Start9 Server is a fork called **Vaultwarden.**  It works perfeclty with all of the apps/integrations that are built for and still named after **Bitwarden.**  We apologize for any confusion this may cause.
 
-    .. youtube:: YcxxVHpm9j0
-        :width: 100%
-
-Follow the guides below to get the most out of your Vaultwarden server.
+Follow the guides below to get the most out of your Vaultwarden server:
 
 .. toctree::
```

Comment:
Something weird happens here - there are 3 links on index page, first 2 go to the same place, probably just need the 2

Reply:
I disagree - the second one goes further down the page to client setup specifically. I think it's fine to do, makes thing much clearer from a user perspective. They are looking for client setup, not initial setup. Those two things should be on the same page as they are but that doesn't mean we want to call that page "initial setup and client setup" - seems like they can be good to go partially setup and a few months later want to setup a different client device - then they have that link right there. I like but can just make it all one link called "Initial Setup & Client Setup" if you like.

Reply:
I would not have put them on the same page.  I don't like the idea of moving to collapsed guides as a crutch.  It's very annoying to navigate once you've opened one.

Reply:
@MattDHill agrees, going to try out subnav instead.